docs/adr/0001-retiring-licence-finder.md

```diff
@@ -0,0 +1,82 @@
+
+## Context
+
+There is a legal requirement for 900 licences need to be created and updated on GOV.UK in the next few months.
+Currently licences are a mainstream document type and the responsibility for maintaining them rests with GDS content designers in Mainstream Publisher.
+
+Developers are also needed to add licences to the search results Licence Finder. Licences are added to this tool by updating a CSV file with the [licence information], e.g. title, description, link, licence identifier and running a rake task. Licences are tagged to [sectors] and [activities] by adding the mapping in a CSV file and running a rake task.
+
+When rendered, licence pages resemble transaction pages, and have an "Apply now" button that takes the user to one of two places: a licence managed by a competent authority on an external website (e.g. TV Licence) or a postcode lookup that allows users to apply for the licence via their local authority. These local authority licences are managed in [Licensify].
+
+Currently users can either find licences in the tool by narrowing down a list of "sectors" and "activities" or by performing a keyword search. Regardless of the approach, the list of results shown to the user is found by querying elastic search with the unique licence identifier assigned to each licence.
+
+If the licence is found in elastic search a link to the licence is displayed to the user.
+
+If the licence cannot be found in elastic search only the title of the licence is displayed to the user. This title may not match the title in the content item.
+
+### Problems with the current approach
+
+Only a few licences can be considered "mainstream". Most are for a specialist activity. However as licences are managed in Mainstream Publisher only GDS content designers can create licence pages.  As Content Designers across government cannot create licence pages, they have taken to using other Whitehall formats to create licence content. This means that the licences created in other formats do not have a licence identifier, and therefore do not appear in the licence-finder results.
+
+Another reason that not all licences appear in the licence finder results is because a developer is needed to updated the CSV files and run a rake task, a step that is easy to miss.
+
+It is possible for no results to be displayed to the user if they do a text search. For example, "restaurant" doesn't yield any results, even though it is in the name of a sector.
+
+Design-wise, the existing tool doesn't match any of the other design patterns on GOV.UK (e.g. smart-answers or finders), nor does it use any design system components.
+
+Finally, the existing tool doesn't contain any extra analytics tracking on the search field or the [sectors] and [activities] list making it difficult to track user behaviour.
+
+## Decision
+
+This Licence Finder application will be retired and replaced with a specialist finder similar to the [Air Accidents Investigation Branch reports] finder.
+
```

Comment:
Is it worth adding something around what facets this finder would use? And if not known call that out as a decision yet to be made?